cypress/integration/podcasts.spec.js

```diff
@@ -16,8 +17,6 @@ describe("newsdesk logged in", () => {
     cy.visit(homeurl)
     cy.get("#login").click()
     cy.contains("Peach Otter").click()
-    cy.get("h2").should("have.text", "⬆️ select a podcast ⬆️")
-    switchPodCast("dev sandbox")
```

Comment:
the context of all these tests is within a podcast, so you may as well leave the switch in